brand/models/brand.py

```diff
@@ -22,54 +22,63 @@ class Brand(TimeStampedModel):
     """
 
     name = models.CharField(
-        "Name of this brand/data source",
-        max_length=200,
-        null=False,
-        blank=False,
-        default="-unnamed-",
+        "Name of this brand", max_length=200, null=False, blank=False, default="-unnamed-"
     )
+    name_locked = models.BooleanField(default=False)
     description = models.TextField(
-        "Description of this instance of this brand/data source",
-        null=True,
-        blank=True,
-        default="-blank-",
+        "Description of this instance of this brand", null=True, blank=True, default="-blank-"
     )
+    description_locked = models.BooleanField(default=False)
     website = models.URLField(
-        "Website of this brand/data source. i.e. bankofamerica.com", null=True, blank=True
+        "Website of this brand. i.e. bankofamerica.com", null=True, blank=True
     )
+    website_locked = models.BooleanField(default=False)
     countries = CountryField(
         multiple=True, help_text="Where the brand offers retails services", blank=True
     )
+    countries_locked = models.BooleanField(default=False)
     tag = models.CharField(
         max_length=100,
         null=False,
         blank=False,
         editable=True,
         unique=True,
-        help_text=(
-            "the tag we use or this brand/datasource record at Bank.Green. ",
-            "Prepend this with the relevant datasource. i.e. banktrack_bank_of_america. "
-            "for brands, prepend with nothing at all i.e. bank_of_america",
-        ),
+        help_text=("the tag we use or this brand record at Bank.Green. ",),
     )
+    tag_locked = models.BooleanField(default=False)
```

Comment:
This should probably be default true. Once a brand is created it has a tag and may begin appearing on the website. We wouldn't want an updated brand name to result in a different url

Reply:
done